driver-core/src/main/java/com/datastax/driver/core/KeyspaceMetadata.java

```diff
@@ -262,21 +286,61 @@ public String exportAsString() {
 
         sb.append(asCQLQuery()).append('\n');
 
-        for (UserType udt : userTypes.values())
+        // include types, tables, views, functions and aggregates, each ordered by name, with one small exception
+        // being that user types are ordered topologically and then by name within same level.
+        for (UserType udt : getSortedUserTypes())
```

Comment:
There already isn't a consistent ordering in the collection itself depending on the order of UDT creation (if created within the same Cluster instance), but they are always ordered topologically (just not consistently).  I think it is not important for it to be sorted at that time, but only when exporting it to cql.  I'll add a separate commit that backs that out, but we can always remove that commit if we want to keep it.   I think the order isn't significant (nor is it documented) for the actual collection members, but it is significant when exporting as a cql string because if it's not ordered correctly it's not valid.